src/util/sql/transform-special-needs.sql

```diff
@@ -5,7 +5,7 @@
 DROP TABLE IF EXISTS ${migrationSchema:name}.evaka_assistance_need;
 CREATE TABLE ${migrationSchema:name}.evaka_assistance_need (
     id UUID PRIMARY KEY DEFAULT ${extensionSchema:name}.uuid_generate_v1mc(),
-    effica_ssn TEXT NOT NULL,
+    effica_ssn TEXT,
```

Comment:
I think the real fix to change line 19: `child.effica_ssn` -> `sn.personid AS effica_ssn`.

And exactly same fix should be applied to `transform-special-means.sql` file.